### ISSUE TITLE
pluggables created and start only in mounted state

### ISSUE THREAD

**Issue by chrisdrew1** [biemond/biemond-oradb#242]
When creating a container and pluggable the pluggables start in mounted state.  I don't see a command/feature to have these set to open read/write in the code.  Am I missing it, or can this be added.

**Comment by biemond** [comment]
Hi, 

that is normal and oradb won't do it. you need to use sqlplus (I don't have custom sqlplus types) , you can decide what to do, see https://www.dummies.com/programming/databases/how-to-create-new-pluggable-databases-in-oracle-12c/

**Comment by chrisdrew1** [comment]
Yeah, my solution is to create an exec statement to open them up and save the state.  tx.

**Comment by biemond** [comment]
my plan was to make a sqlplus provider which checks the current state and opens it. But you can have multiple db on 1 host so I need to read the oratab, set theenv variables and try to connect as sysdba without password.
